### ISSUE TITLE
Feature: Add html version

### ISSUE THREAD

**Issue by hejjoe** [okfde/transparenzgesetz.de#2]
Adding an html version like [this](https://github.com/okfde/gesetz.volksentscheid-transparenz.de/) might help to work better with this complex law.

**Comment by krmax44** [comment]
Yeah, that'd be nice to have! Would you be interested in helping with this feature? The law was written with Word and unfortunately can't be easily converted to Markdown or something like that 🙃
